roles/gitea/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+---
+- name: Register active gitea version
+  ansible.builtin.shell: "set -eo pipefail; /usr/local/bin/gitea -v | cut -d' ' -f 3"
+  args:
+    executable: /bin/bash
+  register: gitea_active_version
+  changed_when: false
+  failed_when: false
+  when: gitea_version_check|bool
+
+- name: Download gitea binary
+  ansible.builtin.get_url:
+    url: "{{ gitea_dl_url }}"
+    dest: /usr/local/bin/gitea
+    owner: root
+    group: root
+    mode: 0755
+    force: true
+  notify: "Restart gitea"
+  when: (not gitea_version_check|bool) or (not ansible_check_mode and (gitea_active_version.stdout != gitea_version))
+
+- name: Create gitea group
+  ansible.builtin.group:
+    name: "{{ gitea_group }}"
+    system: true
+    state: "present"
+
+- name: Create gitea user
+  ansible.builtin.user:
+    name: "{{ gitea_user }}"
+    home: "{{ gitea_home }}"
+    system: true
```

Comment:
Does `system: true` imply it won't have a home directory created? Do we need to set `create_home: false`?

or does it need a home directory?

Reply:
Nevermind, looking below I seemed to answer my own question.